winterfell/webapp/src/js/app.js

```diff
@@ -49,7 +49,7 @@ app.config(function ($routeProvider) {
       templateUrl: "templates/claimView.html"
     })
 
-    .when("/form/:formId", {
+    .when("/form/:claimId/:formId", {
```

Comment:
small nit, but I think it's more RESTful if it is `claim/:claimId/form/:formId`
It's also more consistent with the rest of our endpoints that use claimIds.
Like `/claim/:claimId/dosomething`